tests/integration/lambdas/lambda_integration.py

```diff
@@ -23,6 +22,16 @@
 )
 
 
+# Do not import this function from localstack.utils.common (this is a standalone application / lambda).
```

Comment:
Great idea! We've been thinking of making this Lambda self-contained in a while already.. 👍